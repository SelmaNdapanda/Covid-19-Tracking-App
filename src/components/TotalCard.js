import React from 'react';
import Card from 'react-bootstrap/Card';

const TotalCard = (prop) => {
  const { name, total } = prop;
  return (
    <Card className="border-0">
      <Card.Title className="total-card">{name}</Card.Title>
      <Card.Body className="card-body">
        <Card.Text className="d-flex flex-column">
          <span>{total}</span>
          Total Confirmed Cases
        </Card.Text>
      </Card.Body>
    </Card>
  );
};
export default TotalCard;
