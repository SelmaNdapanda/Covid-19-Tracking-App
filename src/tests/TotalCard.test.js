import { render, screen } from '@testing-library/react';
import TotalCard from '../components/TotalCard';

describe('Card component', () => {
  test('should rendered correctly', () => {
    render(
      <TotalCard name="COUNTRIES" total="638547623" />,
    );
    expect(screen).toMatchSnapshot();
  });
});
