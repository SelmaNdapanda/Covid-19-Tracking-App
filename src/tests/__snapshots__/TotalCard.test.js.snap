// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should rendered correctly 1`] = `
Object {
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "logTestingPlaygroundURL": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
}
`;

exports[`Filter component Snapshot test 1`] = `
<div
  className="border-0 card"
>
  <div
    className="total-card card-title h5"
  >
    COUNTRIES
  </div>
  <div
    className="card-body card-body"
  >
    <p
      className="d-flex flex-column card-text"
    >
      <span>
        638547623
      </span>
      Total Confirmed Cases
    </p>
  </div>
</div>
`;
